subscriptions-api.md

```diff
@@ -453,11 +453,9 @@ Below is an example JSON serialization of a subscription resource:
     "interval": 5
   },
 
-  "filter": {
-    "prefix": {
-      "type": "com.example."
-    }
-  },
+  "filters": [
+    { "prefix": { "type": "com.example." } }
```

Comment:
Do we anticipate more than just `"type": { filter object }` in there? If not, would it make sense to make it:
```
"filters": [
  "prefix": { "type": "com.example." },
  "suffix": { "type": "foo" }
]
```
IOW, do we need the extra {}'s ?